lib/gtl_formatter.rb

```diff
@@ -0,0 +1,107 @@
+class GtlFormatter
+  def self.format_cols(view, row)
+    cols = []
+
+    state = {"state" => :state_format}
+    hardware = {"hardware.bitness" => :hardware_bitness_format}
+    special_cases = {
```

Comment:
Cc @martinpovolny, we now have a list of special cases per report yaml, and per field.
(Except for the *other* set of special cases :))